### DISCUSSION TITLE
Sending prompt as new user / in new incognito window fails

### DISCUSSION THREAD

**Discussion by AInteriorB** [open-webui/open-webui#5377] [General]
# Bug Report

## Installation Method

git clone src installation

## Environment

- **Open WebUI Version:** v0.3.21
- **Ollama (if applicable):** 0.3.10

- **Operating System:** Linux Debian Bookworm
- **Browser (if applicable):** Chrome 128.0.6613.137

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Can submit a prompt as new user or in a new incognito window

## Actual Behavior:

Submitting first prompt fails

## Description

**Bug Summary:**
The javascript error leads to a not usable interface when logging in from a fresh incognito window

## Reproduction Details

**Steps to Reproduce:**
- open new incognito window in chrome
- log-in
- try to send a prompt

## Logs and Screenshots

**Browser Console Logs:**
```
Suggestions.svelte:13 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'reduce')
    at Rk.t.$$.update (Suggestions.svelte:13:4)
    at at (scheduler.js:115:6)
    at ut (scheduler.js:79:5)
```

**Comment by tjbck** [comment]
Unable to reproduce with the latest dev, let us know if you manage to find a fix!
